### DISCUSSION TITLE
Create Selection Rectangle

### DISCUSSION THREAD

**Discussion by AmmarZeer** [xyflow/xyflow#2779] [Q&A]
When selecting nodes using Shift key, a purple rectangle will appear surrounding the selected nodes.

Is there a way to make this rectangle appear when I select the nodes without Shift key, an example would be setting specific nodes `selected` property to `true`, after the setNodes is done I would like to make this rectangle appear to surround the nodes I've selected.

**Comment by Max-ChenFei** [comment]
Hi, I check the doc, and there is no option for this and even I don't want to show the rectangle after finish selection

**Reply by bcakmakoglu** [reply]
You could just hide it with css

```css
.react-flow__nodesselection-rect {
  display: none;
}
```

**Reply by Max-ChenFei** [reply]
Thanks a lot 👍

**Comment by MatheusRoichman** [comment]
@moklick please take a look at this, experiencing the same issue. I'm using the multiSelectionKeyCode and I want the selection rectangle to appear wrapping the selected elements.
